samples/AudioEcsSample/main.cpp

```diff
@@ -93,50 +93,52 @@ int main()
-
-        for (auto [entity, transform] : transforms)
+    scheduler.RegisterSystem(
+        [&uwuBounds, &uwuFlip](auto delta, auto catalogue)
         {
-            TransformComponent newComponent{};
-            newComponent.positionAndLayer = NovelRT::Maths::GeoVector3F::zero();
-            if (uwuFlip)
+            ComponentView<TransformComponent> transforms = catalogue.template GetComponentView<TransformComponent>();
+
+            for (auto [entity, transform] : transforms)
             {
-                newComponent.positionAndLayer.x = 2;
-                uwuBounds = NovelRT::Maths::GeoBounds(
-                    NovelRT::Maths::GeoVector2F(transform.positionAndLayer.x + 2, transform.positionAndLayer.y),
-                    uwuBounds.size, transform.rotationInRadians);
-                if (transform.positionAndLayer.x >= 480)
+                TransformComponent newComponent{};
+                newComponent.positionAndLayer = NovelRT::Maths::GeoVector3F::Zero();
+                if (uwuFlip)
                 {
-                    uwuFlip = false;
+                    newComponent.positionAndLayer.x = 2;
+                    uwuBounds = NovelRT::Maths::GeoBounds(
+                        NovelRT::Maths::GeoVector2F(transform.positionAndLayer.x + 2, transform.positionAndLayer.y),
+                        uwuBounds.size, transform.rotationInRadians);
+                    if (transform.positionAndLayer.x >= 480)
+                    {
+                        uwuFlip = false;
+                    }
                 }
-            }
-            else
-            {
-                newComponent.positionAndLayer.x = -2;
-                uwuBounds = NovelRT::Maths::GeoBounds(
-                    NovelRT::Maths::GeoVector2F(transform.positionAndLayer.x - 2, transform.positionAndLayer.y),
-                    uwuBounds.size, transform.rotationInRadians);
-                if (transform.positionAndLayer.x <= -480)
+                else
                 {
-                    uwuFlip = true;
+                    newComponent.positionAndLayer.x = -2;
+                    uwuBounds = NovelRT::Maths::GeoBounds(
+                        NovelRT::Maths::GeoVector2F(transform.positionAndLayer.x - 2, transform.positionAndLayer.y),
+                        uwuBounds.size, transform.rotationInRadians);
+                    if (transform.positionAndLayer.x <= -480)
+                    {
+                        uwuFlip = true;
+                    }
                 }
+                newComponent.scale = NovelRT::Maths::GeoVector2F::Zero();
+                newComponent.rotationInRadians = 0;
+                transforms.PushComponentUpdateInstruction(entity, newComponent);
             }
-            newComponent.scale = NovelRT::Maths::GeoVector2F::zero();
-            newComponent.rotationInRadians = 0;
-            transforms.PushComponentUpdateInstruction(entity, newComponent);
-        }
-    });
+        });
```

Comment:
Did clang-format from our CI enforce this?

Reply:
it was visual studio and code also doesn't want to change it to how it is in the patch, i don't really get it but it was a bit too much to puzzle out at the time.